client/installation-helper.cc

```diff
@@ -234,6 +276,8 @@ main(int argc, char** argv)
     string root_prefix = "/";
     string default_subvolume_name;
     string description;
+    string snapshot_type = "single";
+    unsigned int pre_num;
```

Comment:
Please initialise to 0 for consistent behaviour if the option "pre-num" is not provided.